CHANGELOG.md

```diff
@@ -2,11 +2,40 @@
 
 ## Unreleased
 
+## 3.13.0 (Feature release)
+
+### New features
+
+#### Add a 'none' option and 'or' divider to checkboxes
+
+You can now add a 'none' option to checkboxes. For example, 'None of the above' or 'None of these options apply to me'.
+
+Use the 'none' option to [allow users to state that none of the checkboxes apply to them](https://design-system.service.gov.uk/components/checkboxes/#add-an-option-for-none-). Otherwise, users would have to work out that they should leave all the checkboxes unticked.
+
+You can now also add an 'or' divider before the 'none' option. Use the 'or' divider to make the 'none' option more visible to users.
+
+This was added in [pull request #2151: Add 'None of the above' and 'or' divider to checkboxes](https://github.com/alphagov/govuk-frontend/pull/2151).
```

Comment:
```suggestion
This was added in [pull request #2151: Add 'None of the above' and 'or' divider to checkboxes](https://github.com/alphagov/govuk-frontend/pull/2151). Thanks to @frankieroberto for contributing this improvement.
```

Reply:
We should make the link explicit as we do with `@natcarey` below:

```suggestion
This was added in [pull request #2151: Add 'None of the above' and 'or' divider to checkboxes](https://github.com/alphagov/govuk-frontend/pull/2151). Thanks to [@frankieroberto](https://github.com/frankieroberto] for contributing this improvement.
```

Reply:
Thanks @lfdebrux ⬆️  looks good I'm happy to commit it.